src/views/admin/PersonImportView.vue

```diff
@@ -0,0 +1,503 @@
+<script setup lang="ts">
+  import FormWrapper from '@/components/form/FormWrapper.vue';
+  import FormRow from '@/components/form/FormRow.vue';
+  import LayoutCard from '@/components/cards/LayoutCard.vue';
+  import {
+    onBeforeRouteLeave,
+    useRouter,
+    type NavigationGuardNext,
+    type RouteLocationNormalized,
+    type Router,
+  } from 'vue-router';
+  import { useRollen, type TranslatedRolleWithAttrs } from '@/composables/useRollen';
+  import { usePersonenkontextStore, type PersonenkontextStore } from '@/stores/PersonenkontextStore';
+  import { useSchulen } from '@/composables/useSchulen';
+  import { computed, onMounted, ref, watch, type ComputedRef, type Ref } from 'vue';
+  import type { TranslatedObject } from '@/types';
+  import { useForm, type BaseFieldProps, type FormContext, type TypedSchema } from 'vee-validate';
+  import { toTypedSchema } from '@vee-validate/yup';
+  import { mixed, object, string } from 'yup';
+  import { useI18n, type Composer } from 'vue-i18n';
+  import { OrganisationsTyp, useOrganisationStore, type OrganisationStore } from '@/stores/OrganisationStore';
+  import { type ImportStore, useImportStore } from '@/stores/ImportStore';
+  import { RollenArt } from '@/stores/RolleStore';
+  import SpshAlert from '@/components/alert/SpshAlert.vue';
+
+  const organisationStore: OrganisationStore = useOrganisationStore();
+  const importStore: ImportStore = useImportStore();
+  const personenkontextStore: PersonenkontextStore = usePersonenkontextStore();
+
+  const router: Router = useRouter();
+  const { t }: Composer = useI18n({ useScope: 'global' });
+
+  const allRollen: ComputedRef<TranslatedRolleWithAttrs[] | undefined> = useRollen();
+  const lernRollen: ComputedRef<TranslatedRolleWithAttrs[] | undefined> = computed(() => {
```

Comment:
Für die Rollen haben wir keinen richtigen Autocomplete hier oder? Die Rollen liegen nur im FE.

Reply:
Ja, die liegen nur im FE, werden aber durch die computed immer wieder aktualisiert. Meinst du ein Select wäre besser?

Reply:
Brauchen wir kein backend autocomplete, weil die Rollen so wenig sind?

Reply:
Das kann ich nicht einschätzen. Christian sagte, es sind zu Beginn nur 13 Rollen und in diesem Feld haben wir sogar nur die Lern-Rollen, was noch mal weniger sind. Ich frage ihn mal, was er für sinnvoll hält.

Reply:
Wenn die Rollen auf <25 begrenzt sind dann würde Ich vor dem go live hier kein autocomple mehr einbauen, damit wir das schnell auf main kriegen.

Reply:
Die Vuetify-Komponente lassen wir als autocomplete, die Funktion mit autocomplete vom BE machen wir in einem separaten Ticket, aus dem von dir genannten Grund. Ist mit PO abgesprochen.